config/default/kustomization.yaml

```diff
@@ -10,7 +10,7 @@ namePrefix: capoci-
 
 # Labels to add to all resources and selectors.
 commonLabels:
-  cluster.x-k8s.io/v1beta1: v1beta1
+  cluster.x-k8s.io/provider: "infrastructure-oci"
```

Comment:
don't we still want to add the version label?

Reply:
as discussed, the version labels is added in the crd/kustomization.yaml folder